### PR TITLE
Enable asimage on Windows (and set correct AFTERIMAGE_INCLUDE_DIR)

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

File: cmake/modules/SearchInstalledSoftware.cmake [hunk 1]
```diff
@@ -395,6 +395,7 @@ if(builtin_afterimage)
       INSTALL_COMMAND  ${CMAKE_COMMAND} -E copy_if_different libAfterImage.lib <INSTALL_DIR>/lib/.
       LOG_DOWNLOAD 1 LOG_CONFIGURE 1 LOG_BUILD 1 LOG_INSTALL 1 BUILD_IN_SOURCE 1
       BUILD_BYPRODUCTS ${AFTERIMAGE_LIBRARIES})
+    set(AFTERIMAGE_INCLUDE_DIR ${CMAKE_BINARY_DIR}/AFTERIMAGE-prefix/src/AFTERIMAGE)
```
Comment (amadio) [inline cmake/modules/SearchInstalledSoftware.cmake]: Why is this needed? Could you please explain? Thanks.

Comment (bellenot) [reply cmake/modules/SearchInstalledSoftware.cmake]: asimage is not installed on Windows, so the include path must be set to its build location (it is only used at build time anyway)

**Comment by bellenot** [comment]
@phsft-bot build just on windows10/vc15
